www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -32,276 +32,152 @@ development. For a comparison of the two development paths, see the
-Android versions earlier than API level 10, and the 3.x versions (Honeycomb,
-API levels 11-13) fall significantly below that 5% threshold.
-
-## Install Cordova Shell Tools
-
-If you want to use Cordova's Android-centered shell tools in
-conjunction with the SDK, download Cordova from
-[cordova.apache.org](http://cordova.apache.org). Otherwise ignore this
-section if you plan to use the cross-platform CLI tool described in
-[The Command-Line Interface](../../cli/index.html).
 
-The Cordova download contains separate archives for each platform. Be
-sure to expand the appropriate archive, `android` in this case, within
-an empty directory.  The relevant executible utilities are available
-in the top-level `bin` directory. (Consult the __README__ file if
-necessary for more detailed directions.)
+#### Installing the Android SDK
 
-These shell tools allow you to create, build, and run Android apps.
-For information on the additional command-line interface that enables
-plugin features across all platforms, see Using Plugman to Manage
-Plugins. See Application Plugins for details on how to develop
-plugins.
-
-## Install the Java Development Kit (JDK)
-
-Install [Java Development Kit (JDK) 7](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html)
-or later. 
-
-When installing on Windows you also need to set `JAVA_HOME` Environment Variable
-according to JDK installation path (for example, C:\Program Files\Java\jdk1.7.0_75).
-
-## Install the Android SDK
-
-Install the [Android Stand-alone SDK Tools](http://developer.android.com/sdk/installing/index.html?pkg=tools) or [Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio). Procceed with `Android Studio` if you plan 
+Install the [Android Stand-alone SDK Tools](http://developer.android.com/sdk/installing/index.html?pkg=tools) or [Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio).
+Proceed with Android Studio if you plan on
 developing new Cordova for Android plugins or using native tools to
-run and debug Android platform. Otherwise, `Android Stand-alone SDK Tools`
-are enough to build and deploy Android application.
+run and debug the Android platform. Otherwise, the Android Stand-alone SDK Tools
+are enough to build and deploy Android applications.
 
 Detailed installation instructions are available as part of installation
 links above.
 
-For Cordova command-line tools to work, or the CLI that is based upon
-them, you need to include the SDK's `tools` and `platform-tools`
-directories in your `PATH`.  On a Mac or Linux, you can use a text editor to
-create or modify the `~/.bash_profile` file, adding a line such as the
-following, depending on where the SDK installs:
+#### Update your PATH
+
+For Cordova's CLI and command line tools to work correctly, you will need to
+include the SDK's `tools` and `platform-tools` directories in your `PATH`.
```

Comment:
The same as for `JAVA_HOME` - this should be [set by `check_reqs` automatically](https://github.com/apache/cordova-android/blob/master/bin/lib/check_reqs.js#L166)